clients/src/main/java/org/apache/kafka/clients/admin/ConfigEntry.java

```diff
@@ -78,8 +78,8 @@ public ConfigEntry(String name, String value, boolean isDefault, boolean isSensi
      * @param isReadOnly whether the config is read-only and cannot be updated
      * @param synonyms Synonym configs in order of precedence
      */
-    ConfigEntry(String name, String value, ConfigSource source, boolean isSensitive, boolean isReadOnly,
-                List<ConfigSynonym> synonyms) {
+    public ConfigEntry(String name, String value, ConfigSource source, boolean isSensitive, boolean isReadOnly,
+                       List<ConfigSynonym> synonyms) {
```

Comment:
This is a public API, so let's not change it here.  If you need a `ConfigEntry` with synonyms, create a static method in `ConfigTest` that will make it.  That will keep the change confined to the test code.

Reply:
Done.